_docs_integrate/data-model-overview.md

```diff
@@ -752,15 +757,15 @@ Internally, the succeeded version will then be created at your side as successor
 | successorId      | `string`                                                                                       | The `id` of the LocalAttribute it was succeeded by |
 | successorContent | [`IdentityAttribute`](#identityattribute) \| [`RelationshipAttribute`](#relationshipattribute) | The updated `content` of the LocalAttribute        |
 
-#### ThirdPartyOwnedRelationshipAttributeDeletedByPeerNotificationItem
+#### ThirdPartyRelationshipAttributeDeletedByPeerNotificationItem
 
-If you have shared a RelationshipAttribute, that is not owned by you, with a peer from a different Relationship and that peer deletes their third party owned RelationshipAttribute, a ThirdPartyOwnedRelationshipAttributeDeletedByPeerNotificationItem will be sent to you.
-Internally, for your corresponding third party owned RelationshipAttribute the `deletionInfo.deletionStatus` will be set to `"DeletedByPeer"`.
+If your peer deletes an [emitted or received ThirdPartyRelationshipAttribute]({% link _docs_integrate/attribute-introduction.md %}#emitted-and-received-thirdpartyrelationshipattributes), a ThirdPartyRelationshipAttributeDeletedByPeerNotificationItem will be sent to you.
+Internally, for your corresponding ThirdPartyRelationshipAttribute the `deletionInfo.deletionStatus` will be set to `"DeletedByPeer"`.
```

Comment:
If the `owner` deletes the Attribute, is `"DeletedByOwner"` also possible? 🤔

Reply:
Currently we don't distinguish between owner and peer or emitter and recipient. However, there is an open task regarding the deletion of ThirdPartyRelationshipAttributes and this might be adjusted in this context.